e2e/validators.go

```diff
@@ -368,3 +386,23 @@ func containerdWasmShimsValidator() *LiveVMValidator {
 		},
 	}
 }
+
+func KubeletHasNotStoppedValidator() *LiveVMValidator {
```

Comment:
It'd be nice to have a comment about why we don't want kubelet to be restarted - the implications of that. So that the next dev doesn't delete the test and understands the issue.